utoipa-gen/src/path/media_type.rs

```diff
@@ -90,28 +95,56 @@ impl<'m> MediaTypeAttr<'m> {
     }
 
     pub fn parse_named_attributes(
-        media_type: &mut MediaTypeAttr,
+        &mut self,
         input: ParseStream,
         attribute: &Ident,
     ) -> syn::Result<()> {
         let name = &*attribute.to_string();
 
         match name {
             "example" => {
-                media_type.example = Some(parse_utils::parse_next(input, || {
+                self.example = Some(parse_utils::parse_next(input, || {
                     AnyValue::parse_any(input)
                 })?)
             }
             "examples" => {
-                media_type.examples = parse_utils::parse_comma_separated_within_parenthesis(input)?
+                self.examples = parse_utils::parse_comma_separated_within_parenthesis(input)?
+            }
+            "encoding" => {
+                struct KV {
+                    k: String,
+                    v: Encoding,
+                }
+
+                impl Parse for KV {
+                    fn parse(input: ParseStream) -> syn::Result<Self> {
+                        let key_val;
+
+                        syn::parenthesized!(key_val in input);
+
+                        let k = key_val.parse::<syn::LitStr>()?.value();
+
+                        key_val.parse::<Comma>()?;
```

Comment:
I think instead of comma, this could be `=` sign. This would then match better with the syntax already at present in elsewhere e.g. links.

Reply:
Sounds good. In that case I'll go with the `parse_next` functions from the utils.